### DISCUSSION TITLE
Sorting eme_bookings by date

### DISCUSSION THREAD

**Discussion by TimB111** [liedekef/events-made-easy#168] [General]
I'm using eme_bookings to give the bookings list to an event host who doesn't have access to the back end but still needs to have access to the live list as people book up to the event start time, and even after it has started. It produces the list of bookings sorted by last name. In the documentation I don't see an option to sort by booking date. Is this possible?

**Comment by liedekef** [comment]
There currently is no order option for this shortcode. But this changeset (for the next version) will help:

https://github.com/liedekef/events-made-easy/commit/d3801c9aec9df152a0dc8130fbbb84decf7018ce

**Reply by TimB111** [reply]
That's great, thank you!
